launch/client.py

```diff
@@ -448,6 +449,7 @@ def create_model_endpoint(
         per_worker: int = 1,
         gpu_type: Optional[str] = None,
         endpoint_type: str = "sync",
+        post_inference_hooks: Optional[List[PostInferenceHooks]] = None,
```

Comment:
should we make this empty array just in case someone breaks the default in the server side?

Reply:
Good article on why default should be None not empty list: https://satran.in/b/python--dangerous-default-value-as-argument

Tl;dr is that if you have the default as `[]`, it's considered unsafe since the interpreter could treat the list as the same reference between **different calls** to the method.

Reply:
(Actually I'm not that pro and didn't know that) - `pylint` caught it for me 😆